script.js

```diff
@@ -303,3 +311,53 @@ function appendStory(story, board, storyUrl, statusBadge, estimate) {
     "</div>"
   );
 }
+
+function initializeDragAndDrop () {
+  $(".js-board").sortable({
+    connectWith: ".js-board",
+    stop: function(event, ui) {
+      updatePivotal(ui);
+    }
+  });
+} //end initializeDragAndDrop
+function updatePivotal(ui) {
+    let id = ui.item.data("id");
+    let newStatus = ui.item.closest(".js-board").data("status");
+    let afterId = ui.item.prev().data("id");
+    let beforeId = ui.item.next().data("id");
+    let urlId = "/" + ui.item.data("id");
+    let currentCard = ui.item;
+    if (beforeId === undefined) beforeId = null;
+    if (afterId === undefined) afterId = null;
+    if (newStatus === 'accepted') {
+      instance.put("/" + id, {
+      "current_state": newStatus,
+      })
+      .then(function(response) {
+        currentCard.find(".status-badge").removeClass()
+        .addClass("status-badge " + response.data.current_state)
+        .text(response.data.current_state);
+        
+      })
+      .catch(function(error) {
+      console.log(error);
+      });
+    } else {
+      instance.put("/" + id, {
+        "current_state": newStatus,
+        "after_id": afterId,
+        "before_id": beforeId
```

Comment:
If both `after_id` and `before_id` are `nil`, would it work to pass them along for an accepted story? Then we might be able to get rid of this duplicated branch code.

Reply:
Even if they are null, if i pass `before_id` and `after_id`  i get PUT error from Pivotal :(